emscripten.py

```diff
@@ -37,14 +37,45 @@
 WASM_INIT_FUNC = '__wasm_call_ctors'
 
 
+def make_wasm_table_static_dyncaller(func):
```

Comment:
It looks like this is only used in minimal runtime?   Don't we want to always generate these helpers on the JS side (when the user asks for them or needs them)?

Reply:
Yeah, the intent is not to be minimal runtime only, but I just started looking at this in minimal runtime first. This may probably not be needed after all, I'll delete it soon if not.